src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { Filter } from '../../types/Filter';
+
+interface Props {
+  filter: Filter;
+  setFilter: (filter: Filter) => void;
+  activeTodosCount: number;
+  areThereCompletedTodos: boolean;
+}
+
+export const Footer: React.FC<Props> = ({
+  filter,
+  setFilter,
+  activeTodosCount,
+  areThereCompletedTodos,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {activeTodosCount} items left
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
```

Comment:
Use Object.values(your created enum) and render these options with map() method

Reply:
Not fixed 
<img width="963" alt="image" src="https://github.com/user-attachments/assets/731bf58a-467e-47c5-8742-8c9a135db1ba">